pyscal/pyscallist.py

```diff
@@ -68,6 +69,104 @@ def append(self, pyscal_obj):
             raise ValueError
         self.pyscal_list.append(pyscal_obj)
 
+    def df(self):
+        """Dump dataframes of generated relperm data
+
+        Column names are compatible with ecl2df.satfunc. Always uppercase
+        and capillary pressure is PCOW or PCOG (wateroil vs gasoil)
+
+        If the PyscalList contains SCALrecommendations, the CASE column
+        will contain the strings 'pess', 'base' and 'opt' (independent of
+        any alias name potentially used in an input xlsx/csv)
+
+        Returns:
+            pd.DataFrame
+        """
+        gasoil_col_renamer = {"sg": "SG", "krg": "KRG", "krog": "KROG", "pc": "PCOG"}
+        wateroil_col_renamer = {"sw": "SW", "krw": "KRW", "krow": "KROW", "pc": "PCOW"}
```

Comment:
Since there are no overlapping columns between gasoil and wateroil, should we consider to merge them on SATNUM,  CASE if self.pyscaltype == SCALrecommendation? Would half the number of rows and remove NaNs. Introduces a risk that a user combines two incompatible columns though.

Reply:
A little bit skeptical, as data in a dataframe-row usually means that those data are related somehow, which would not be the case here. Afraid that that will cause some confusion. The number of saturation points for WaterOil and GasOil might not match either, so some NaNs are expected.

It would solve a NaN-issue, and also type-inferral issue when loading from CSV (for those parsers that do only look for the say first 100 rows for type inferral)

It should also be followed by a similar change in `ecl2df.satfunc`

Reply:
Then leave it as is, I agree that the risk of confusion is definitely there